Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -678,19 +758,30 @@ final class RecoveryTests: XCTestCase {
         DiagnosticSpec(locationMarker: "2️⃣", message: "all statements inside a switch must be covered by a 'case' or 'default' label", fixIts: ["insert label"]),
         DiagnosticSpec(locationMarker: "3️⃣", message: "consecutive statements on a line must be separated by ';'", fixIts: ["insert ';'"]),
         DiagnosticSpec(locationMarker: "4️⃣", message: "'case' can only appear inside a 'switch' statement or 'enum' declaration"),
-      ]
+      ],
+      fixedSource: """
+        switch {
+        case <#identifier#>: 42 }(); {
+          case _: return
+        }
+        """
     )
   }
 
-  //===--- Recovery for missing braces in nominal type decls.
+  //MARK: - Recovery for missing braces in nominal type decls.
```

Comment:
```suggestion
  // MARK: - Recovery for missing braces in nominal type decls.
```

(same for the other `MARK`s)